INPUT-OUTPUT-FORK.md

```diff
@@ -0,0 +1,13 @@
+# Reasons for the fork
+
+Here we expose the reasons for the Input Output Global fork `lmdb-simple`.
+
+A dedicated Hackage instance called [Cardano Haskell Packages
```

Comment:
There are more reasons for this fork, namely:
- Don't use the serialise instances
- The cursor improvements

Reply:
I've added this to the text